rust/fory-derive/src/util.rs

```diff
@@ -23,13 +23,25 @@ pub fn sorted_fields(fields: &Fields) -> Vec<&Field> {
 }
 
 pub fn get_sorted_fields<'a>(fields: &[&'a Field]) -> Vec<&'a Field> {
-    use crate::object::util::get_sorted_field_names;
+    use crate::object::util::{get_sorted_field_names, is_tuple_struct};
 
+    // For tuple structs, we must preserve the original field order
```

Comment:
@ariesdevil If we take tuple struct as named struct with name as `0/1/2/3/...`, the `get_sorted_field_names` should work too?

I'm think whether we could use same protocol as named struct. In this way, the compatible mode will work with minimal changes